# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../elinks/elinks.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

2006-04-02: we have to compile elinks without NLS support because of conflicts with glibc

--- elinks-0.11.1/src/intl/gettext/Makefile.orig	2006-04-02 11:33:24.000000000 +0200
+++ elinks-0.11.1/src/intl/gettext/Makefile	2006-04-02 11:36:39.000000000 +0200
@@ -26,9 +26,9 @@
 
 
 # $(builddir)/charset.alias: $(srcdir)config.charset
-$(builddir)/charset.alias: $(srcdir)config.charset
-	$(SHELL) $(srcdir)config.charset '$(host)' > $@.new
-	mv $@.new $@
+#$(builddir)/charset.alias: $(srcdir)config.charset
+#	$(SHELL) $(srcdir)config.charset '$(host)' > $@.new
+#	mv $@.new $@
 
 # FIXME: Building plural.c from plural.y on the fly doesn't work
 
@@ -40,8 +40,8 @@
 	$(YACC) $(YFLAGS) --output $@ $<
 	rm -f $*.h
 
-install-local: $(builddir)/charset.alias all
-	@if test '$(CONFIG_NLS)' = yes; then \
+install-local: all
+	@if test '$(CONFIG_NLS)' = no; then \
 	  temp=$(DESTDIR)$(libdir)/t-charset.alias; \
 	  dest=$(DESTDIR)$(libdir)/charset.alias; \
 	  if test -f $(DESTDIR)$(libdir)/charset.alias; then \
